quickstarts/hello-rules/hellorules/pom.xml

```diff
@@ -24,7 +24,7 @@
   <properties>
     <jaxb-api.version>2.3.0.redhat-1</jaxb-api.version>
     <maven.compiler.version>3.8.0</maven.compiler.version>
-    <ba-platform-bom.version>8.0.4.redhat-00003</ba-platform-bom.version>
+    <ba-platform-bom.version>8.0.5.redhat-00001</ba-platform-bom.version>
     <org.jboss.bom.version>7.4.10.GA</org.jboss.bom.version>
```

Comment:
I believe this bom version is taken from EAP, so you should update it to 7.4.14.GA which is the version of EAP we are using for this release.

Please also update it in the other pom files.

Reply:
I have updated the pom.xml with correct EAP version.